Makefile

```diff
@@ -32,11 +32,10 @@ container-build:
 	golang:1.15 make binary
 
 crosscompile:
-	mkdir -p bin/linux/{arm,arm64,amd64} bin/darwin/amd64
-	$(MAKE) container-build OS=linux ARCH=amd64 && mv bin/openapi-generator-go bin/linux/amd64/
-	$(MAKE) container-build OS=linux ARCH=arm && mv bin/openapi-generator-go bin/linux/arm/
-	$(MAKE) container-build OS=linux ARCH=arm64 && mv bin/openapi-generator-go bin/linux/arm64/
-	$(MAKE) container-build OS=darwin ARCH=amd64 && mv bin/openapi-generator-go bin/darwin/amd64/
+	$(MAKE) container-build OS=linux ARCH=amd64 && mv bin/openapi-generator-go bin/openapi-generator-go-linux-amd64
+	$(MAKE) container-build OS=linux ARCH=arm && mv bin/openapi-generator-go bin/openapi-generator-go-linux-arm
+	$(MAKE) container-build OS=linux ARCH=arm64 && mv bin/openapi-generator-go bin/openapi-generator-go-linux-arm64
+	$(MAKE) container-build OS=darwin ARCH=amd64 && mv bin/openapi-generator-go bin/openapi-generator-go-darwin-amd64
```

Comment:
I personally don't like the idea that we build the existing target in a non-deterministic way. I think it would be way better to refactor the current targets to support a parameterised output path.

Reply:
I don't get what you mean, where do you see non-determinism? Regarding the parameterized output path: Before this change it moved the binaries to folders like `bin/linux/amd64/openapi-generator-go` but I found myself  doing the renames everytime I did a release here. So I thought it might be better to just say `make crosscompile` and then have all the artifacts for the upload to the release page with the correct name.